### DISCUSSION TITLE
Why are sponsors "buy tiktok" and "buy traffic"

### DISCUSSION THREAD

**Discussion by AldeRoberge** [requarks/wiki#7441] [Help / Questions] [answered]
![image](https://github.com/user-attachments/assets/a8cac17e-9388-44bf-82c6-39eed0b5da39)

These 3 link to weird websites about buying TikTok and Website traffic... Is that normal?

**Comment by NGPixel** [comment, marked as answer]
The table is auto-populated from whoever donates to the project on OpenCollective. This unfortunately mean poor quality sponsors that just want to put their spam links everywhere.

I just enabled some categories to auto-reject. This should hopefully help reduce this kind of sponsors in the future.
